online-workflow/src/main/java/eu/itesla_project/online/security_rules/ContingencyEvaluator.java

```diff
@@ -106,9 +106,6 @@ private RulesFacadeResults evaluate(String networkId, String stateId, Map<HistoD
         boolean dynamicIndexUnsafe = false;
         if (checkRules) {
             checkRules(rules, invalidRulesIndexes, bacecaseInvalidRulesIndexes);
-            if ("0".equals(stateId)) {
```

Comment:
Why do you remove this block?

Reply:
Reassignement a pararmeter is not allowed and this one is not used after this statement, so i did not created a new XXX xxx = instanceFromParameter(as i do in other case)